src/reddit_analysis.py

```diff
@@ -0,0 +1,48 @@
+from vaderSentiment.vaderSentiment import SentimentIntensityAnalyzer
+
+
+class RedditAnalysis:
+    def __init__(self) -> None:
+        self.results = {}
+        self.stock_count = {}
+        
+    def run_analysis(self, dict):
+        analyzer = SentimentIntensityAnalyzer()
+
+        for sentence in dict:
+            self.sentiment_analyzer_scores(dict[sentence], analyzer)
```

Comment:
sentence is a little bit misleading what about post?

Reply:
I was thinking maybe we can change also dict to be posts and change it fromm data_gatherer too